hopsworks-common/src/main/java/io/hops/hopsworks/common/featurestore/featuregroup/FeaturegroupController.java

```diff
@@ -289,15 +283,33 @@ public FeaturegroupDTO updateFeaturegroupMetadata(
       onDemandFeaturegroupController.updateOnDemandFeaturegroupMetadata(featuregroup.getOnDemandFeaturegroup(),
         (OnDemandFeaturegroupDTO) featuregroupDTO);
     }
-    
+
+    // also in this case we should update the jobs. just in case there is a client relying on the fact that the
```

Comment:
I see your point, we should check soon if this query param is still used anywhere, and otherwise get rid of it all together